libraries/helpers.rb

```diff
@@ -0,0 +1,22 @@
+module RabbitMQCookbook
+  module Helpers
+    require 'mixlib/shellout'
+
+    def remove_self_from_cluster
```

Comment:
Why not `reset_this_node`? I'm not sure what `rmq` contributes in this case. The other function doesn't have `rmq` in the name.